lib/chef/provider/package/macports.rb

```diff
@@ -3,7 +3,7 @@ class Provider
     class Package
       class Macports < Chef::Provider::Package
 
-        provides :macports_package, os: "mac_os_x"
+        provides :macports_package, os: "darwin"
```

Comment:
I sort of feel like we should be more narrow and then expand later if there's a reason, rather than try to do it the other direction since its avoids a possible major revision blocker to the later change.